tools/docker/build_wheel.Dockerfile

```diff
@@ -7,8 +8,9 @@ ARG PY_VERSION
 RUN python$PY_VERSION -m pip install --upgrade pip setuptools auditwheel==2.0.0
 
 COPY tools/install_deps/ /install_deps
+ARG TF_VERSION
```

Comment:
Rationale for two `ARG TF_VERSION`?

Reply:
When using an `ARG` before a from, the ARG is not valid afterwards and needs to be declared again from what I remember. Let me find a piece of docs about that.

Reply:
See https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact